cmd/gitserver/server/customfetch_test.go

```diff
@@ -23,6 +22,9 @@ func TestEmptyCustomGitFetch(t *testing.T) {
 }
 
 func TestCustomGitFetch(t *testing.T) {
+	// mock value for test
+	customGitFetchCmdConf = "/tmp/customGitConfig.json"
```

Comment:
/tmp is not always a writeable tmp dir. Luckily go provides something which will return a path for you and clean it up when the test is done:
```suggestion
	customGitFetchCmdConf = filepath.Join(t.TempDir(), "customGitConfig.json")
```